src/anemoi/training/train/train.py

```diff
@@ -34,13 +34,15 @@
 from anemoi.training.diagnostics.logger import get_wandb_logger
 from anemoi.training.distributed.strategy import DDPGroupStrategy
 from anemoi.training.train.forecaster import GraphForecaster
+from anemoi.training.utils.checkpoint import transfer_learning_loading
 from anemoi.training.utils.jsonify import map_config_to_primitives
 from anemoi.training.utils.seeding import get_base_seed
 
 if TYPE_CHECKING:
     from torch_geometric.data import HeteroData
 
 LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)  # Change DEBUG to INFO, WARNING, etc., as needed
```

Comment:
Needs removing when draft phase over?

Reply:
Yes sure! Here for debugging only

Reply:
In case you want to avoid setting these levels and forgetting, the documentation shows how to set these from the configs:

https://anemoi-training.readthedocs.io/en/latest/user-guide/debugging.html#debug-flags-for-better-error-handling